packages/components/table/stories/table.stories.tsx

```diff
@@ -1038,13 +1033,22 @@ export const CustomWithClassNames = {
     ...defaultProps,
     classNames: {
       base: ["max-w-3xl", "bg-gradient-to-br", "from-purple-500", "to-indigo-900/90", "shadow-xl"],
-      th: ["bg-transparent", "text-white/70", "border-b", "border-white/10"],
+      th: [
+        "bg-transparent",
+        "text-black/70",
+        "dark:text-white/70",
```

Comment:
prefer just use semantic colors (apply to all `dark:*`)

Reply:
used text-default, please check if it looks fine